CIScripts/Test/Tests/DockerDriver.Tests.ps1

```diff
@@ -0,0 +1,74 @@
+Param (
+    [Parameter(Mandatory=$true)] [string] $TestbedAddr,
+    [Parameter(Mandatory=$true)] [string] $ConfigFile
+)
+
+. $PSScriptRoot\..\Utils\CommonTestCode.ps1
+. $PSScriptRoot\..\Utils\ComponentsInstallation.ps1
+. $PSScriptRoot\..\TestConfigurationUtils.ps1
+. $PSScriptRoot\..\..\Common\Aliases.ps1
+. $PSScriptRoot\..\..\Common\VMUtils.ps1
+. $PSScriptRoot\..\PesterHelpers\PesterHelpers.ps1
+
+. $ConfigFile
+#$TestConf = Get-TestConfiguration
+$Session = New-PSSession -ComputerName $TestbedAddr -Credential (Get-TestbedCredential)
+$TestsPath = "C:\Program Files\Juniper Networks\"
+
+function Start-DockerDriverUnitTest {
+    Param (
+        [Parameter(Mandatory=$true)] [PSSessionT] $Session,
+        [Parameter(Mandatory=$true)] [string] $Component
+    )
+
+    $TestFilePath = ".\" + $Component + ".test.exe"
+    $Command = @($TestFilePath, "--ginkgo.noisyPendings", "--ginkgo.failFast", "--ginkgo.progress", "--ginkgo.v", "--ginkgo.trace")
+    $Command = $Command -join " "
+
+    $Res = Invoke-Command -Session $Session -ScriptBlock {
+        Push-Location $Using:TestsPath
+
+        # Invoke-Command used as a workaround for temporary ErrorActionPreference modification
+        $Res = Invoke-Command -ScriptBlock {
+            $ErrorActionPreference = "SilentlyContinue"
+            Invoke-Expression -Command $Using:Command | Write-Host
+            return $LASTEXITCODE

```

Comment:
$LASTEXITCODE may be not set if we invoke command that doesn't exist
There might be situation that previous test set $LASTEXITCODE to 0 and then test might pass despite the fact that it was not invoked correctly.